src/client/entity-editor/author-credit-editor/author-credit-row.tsx

```diff
@@ -80,22 +84,42 @@ function AuthorCreditRow({
 	author,
 	joinPhrase,
 	name,
+	isUf,
 	onAuthorChange,
 	onJoinPhraseChange,
+	onClearHandler,
 	onNameChange,
-	onRemoveButtonClick
+	onRemoveButtonClick,
+	...rest
 }: Props) {
+	const SelectWrapper = !isUf ? EntitySearchFieldOption : SearchEntityCreate;
+	const onChangeHandler = React.useCallback((value, action) => {
+		if (action && ['clear', 'pop-value', 'select-option'].includes(action.action) && author && author.get('__isNew__', false)) {
+			onClearHandler(author.get('id'));
+		}
+		onAuthorChange(value);
+	}, [author]);
+	const handleButtonClick = React.useCallback(() => {
+		// don't remove author if it's first row
```

Comment:
Since AC select field is directly linked to first row, removing it would cause problems hence i disabled the button though now i think about it, it would be better to clear first row (resetting) on remove button click, will do that instead.

Reply:
Yes, I think that would be the right mechanism :+1: